model/host/db.go

```diff
@@ -211,8 +211,12 @@ func runningHostsQuery(distroID string) bson.M {
 	return query
 }
 
-func provisioningHostsQuery(distroID string) bson.M {
-	query := bson.M{StatusKey: bson.M{"$in": []string{evergreen.HostBuilding, evergreen.HostStarting}}}
+func startedTaskHostsQuery(distroID string) bson.M {
+	query := bson.M{
+		StatusKey:   bson.M{"$in": evergreen.StartedHostStatus},
+		UserHostKey: false,
```

Comment:
This gets spawn hosts but not hosts spawned by tasks.  I think you have to either check `h.SpawnOptions.SpawnedByTask` in addition to `UserHostKey` or only check that `h.StartedBy != evergreen.User`. (The latter is what `IsLive()` does.)